src/main/java/com/redhat/devtools/lsp4ij/features/codeBlockProvider/LSPCodeBlockProvider.java

```diff
@@ -0,0 +1,251 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
```

Comment:
2025

Reply:
This class already existed in 2024 and was moved across packages. It should still be 2024.